src/io/flutter/utils/JxBrowserUtils.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.utils;
+
+import com.intellij.openapi.util.SystemInfo;
+
+public class JxBrowserUtils {
+  private static final String JXBROWSER_FILE_PREFIX = "jxbrowser";
+  private static final String JXBROWSER_FILE_VERSION = "7.8";
+  private static final String JXBROWSER_FILE_SUFFIX = "jar";
+
+  public static String getPlatformFileName() {
+    String name = "";
+    if (SystemInfo.isMac) {
+      name = "mac";
+    } else if (SystemInfo.isWindows) {
+      if (SystemInfo.is32Bit) {
+        name = "win32";
+      } else if (SystemInfo.is64Bit) {
+        name = "win64";
+      }
+    } else if (SystemInfo.isLinux && SystemInfo.is64Bit) {
+      name = "linux64";
+    }
+
+    if (name.isEmpty()) {
```

Comment:
are we catching this exception? It would be nice to just fail gracefully if a platform we don't support is used.

Reply:
Ah true. Changed this to a checked exception.